### ISSUE TITLE
BPA TODO rule for Format String Expressions

### ISSUE THREAD

**Issue by edhans** [TabularEditor/TabularEditor3#951]
**Is your feature request related to a problem? Please describe.**
We either need a new "applies to" type, or the object to reference to see if the "TODO" string shows up in a Format String Expression or its description.
![image](https://github.com/TabularEditor/TabularEditor3/assets/45298358/de7e5587-2196-4a09-b7dd-8137956a3305)

**Describe the solution you'd like**
I want to create a BPA rule that will find "TODO" in format string expressions or its description field.

**Describe alternatives you've considered**
None

**Additional context**

**Comment by DBojsen** [comment]
Hi @edhans 

I experimented a little and came up with this:
![image](https://github.com/TabularEditor/TabularEditor3/assets/22237520/3fe88b35-3190-4898-a7c7-1e520dc57f09)

It can succesfully catch the TODO's that I have tested with:
![image](https://github.com/TabularEditor/TabularEditor3/assets/22237520/ec5d1d22-ed7c-41b3-b6dd-51f1d6823103)

Does this do it for you?

BR David - TE3 Support

**Comment by edhans** [comment]
Thanks @DBojsen  - that worked. I had actually tried that and it failed, but it was because I added it to my BPA rule that also included expressions and descriptions for Calculated Columns, Calc Tables, Calc Items, and Named Expressions - it blanked out and I thought I had just guessed the object name wrong. Turns out I guessed right, was just applying it to objects that don't have format string expressions.

Again, thanks!
